### DISCUSSION TITLE
We need to add an upgrade guide to the wiki.

### DISCUSSION THREAD

**Discussion by meotimdihia** [seaweedfs/seaweedfs#3800] [Ideas]
The wiki doesn't answer these questions: 
- What containers should we stop step by step? It should be like https://github.com/seaweedfs/seaweedfs/wiki/Production-Setup
- Can we achieve zero downtime while upgrading?

**Comment by kmlebedev** [comment]
> * Can we achieve zero downtime while upgrading?

Yes.  just helm upgrade

**Reply by meotimdihia** [reply]
I just have used a simple docker installation on the dedicated servers.
